stories/core/Table.stories.tsx

```diff
@@ -2269,3 +2269,63 @@ VirtualizedSubRows.argTypes = {
   isLoading: { control: { disable: true } },
   data: { control: { disable: true } },
 };
+
+export const DraggableColumns: Story<Partial<TableProps>> = (args) => {
+  const onClickHandler = (
+    props: CellProps<{ name: string; description: string }>,
+  ) => action(props.row.original.name)();
+
+  const columns = useMemo(
```

Comment:
It would be nice to have more columns, you could copy from `HorizontalScroll` story but without widths.